ext/storage_sources/azure_store/test/test_azure_connection.cpp

```diff
@@ -31,4 +31,39 @@
 
 #include "azure_connection.h"
 
-TEST_CASE("testing class azure_connection", "azure_connection") {}
+TEST_CASE("testing class azure_connection", "azure_connection")
+{
+
+    std::vector<std::string> objects;
+    azure_connection pfx_test = azure_connection("myblobcontainer1", "pfx_test_");
+
+    // There is nothing in the container so there should be 0 objects
+    objects.clear();
+    REQUIRE(pfx_test.list_objects("", objects, false) == 0);
+    REQUIRE(objects.size() == 1);
+
+    objects.clear();
+    REQUIRE(pfx_test.put_object(
```

Comment:
This object stays in the cloud, make sure to delete it. I suggest using `CHECK` macro instead of `REQUIRE` for all test assertions excluding the case when you delete the object(s) you use.

Reply:
changed from `REQUIRE` to `CHECK`